GitUI/UserControls/FilterToolBar.Designer.cs

```diff
@@ -129,7 +129,7 @@ private void InitializeComponent()
             this.tsmiShowReflogs.Image = global::GitUI.Properties.Images.Book;
             this.tsmiShowReflogs.Name = "tsmiShowReflogs";
             this.tsmiShowReflogs.Size = new System.Drawing.Size(23, 22);
-            this.tsmiShowReflogs.Text = "Reflogs";
+            this.tsmiShowReflogs.Text = "R&eflogs";
```

Comment:
I would like to remove the test label on the button, there is a tooltip

Reply:
Apologies, I can't parse what you're saying.

Reply:
Reflogs should only have an image, no text
Only Commit button has that (filter fields could be modified too)

Reply:
(I took me a while, too: s/test/text/.)
I have removed the text label and updated the screenshot.